compiler/src/dmd/expression.d

```diff
@@ -189,39 +189,6 @@ extern (C++) void expandTuples(Expressions* exps, Identifiers* names = null)
     }
 }
 
-/****************************************
- * Expand alias this tuples.
- */
-TupleDeclaration isAliasThisTuple(Expression e)
```

Comment:
this seems to be an unrelated refactor. I agree with it, but it should be a separate PR.

Reply:
This function uses typesem.toDsymbol. I could make it import dmd.typesem and then make another PR to move it to expressionsem, but I think it's better to just move it now to expressionsem since expression.d does not rely anymore on semantic routines.

Please merge as is.